thermo/gpumd/io.py

```diff
@@ -249,6 +249,61 @@ def import_trajectory(filename='movie.xyz', in_file=None, atom_types=None):
 # Write Related
 #########################################
 
+def create_kpoints(special_points, npoints, atom_c):
+    """
+     Creates the file "kpoints.in", which specifies the kpoints needed for src/phonon
+
+    Args:
+        atom_c (ase.Atom):
+            atom to build
```

Comment:
```suggestion
        atoms (ase.Atoms):
            Atoms used to generate kpoints.in.
```

Reply:
The user should be providing an Atoms object, not Atom. I changed the variable name to match the convention used in ASE.